dl/src/main/scala/com/intel/analytics/bigdl/parameters/AllReduceParameter.scala

```diff
@@ -136,7 +136,9 @@ class AllReduceParameter[T: ClassTag](id: Long, partitionNum: Int,
         override def call(): Int = {
           val blockId = getWeightBlockId(pid)
           val localBuffer = BlockManagerWrapper.byteBufferConvert(
-            bm.getLocalBytes(blockId).getOrElse(bm.getRemoteBytes(blockId).get))
+            bm.getLocalBytes(blockId).getOrElse(bm.getRemoteBytes(blockId)
+                .getOrElse(throw new IllegalArgumentException(s"Can't get the block(${blockId})." +
+                s"Please check if there is executor lost"))))
```

Comment:
Why IllegalArgumentException? And shouldn't we try-catch for all the operations in call(), just as we did for ThreadPool.invokeAndWait?